### ISSUE TITLE
How to hide bottomNavBar on scrolling up and down?

### ISSUE THREAD

**Issue by khaledkhj** [BilalShahid13/PersistentBottomNavBar#127]
Hi,

What would be the best to show/hide the bottomNavBar on scrolling up and down?
As an example for that is LinkedIn app? 

Thank you.

**Comment by BilalShahid13** [comment]
Yes it is possible. There will be an argument in the main navigation bar _PersistentTabView_ widget named hide **hideNavigationBar**. You can toggle it between _true_ or _false_ and each time the value changes, the navigation will appear and disappear with a slide animation. 

Now when to make the navigation bar disappear, that depends on your logic. You can use a combination of _ScrollController_'s listener and a _provider_ but there can be other solutions.

**Comment by khaledkhj** [comment]
Thank you @BilalShahid13 
I tried using Provider like the following:
hideNavigationBar: Provider.of<BottomNavigationBarPoperties>(context) .getIsBottomNavBarHidden
But the issue is that the navigationbar is rebuilding itself when the value changes.
Is there any way to avoid this rebuilding issue?
